plugins/modules/system/parted.py

```diff
@@ -95,6 +95,12 @@
     type: str
     choices: [ absent, present, info ]
     default: info
+  fs_type:
+    description:
+     - If specified, will set filesystem type to given partition
+    type: str
+    default: ext4
```

Comment:
```suggestion
```
Can't this default break current users' pipelines?
Maybe this is not a type that users expected to see after upgrade. See the solution in the main function.

Reply:
I don't think it breaks anything, but I agree to make it optional to emulate `parted` behaviour.